src/main/scala/rocket/RocketCore.scala

```diff
@@ -918,8 +919,8 @@ class Rocket(tile: RocketTile)(implicit p: Parameters) extends CoreModule()(p)
          io.hartid, coreMonitorBundle.timer, coreMonitorBundle.valid,
          coreMonitorBundle.pc,
          Mux(wb_ctrl.wxd || wb_ctrl.wfd, coreMonitorBundle.wrdst, 0.U),
-         Mux(coreMonitorBundle.wren, coreMonitorBundle.wrdata, 0.U),
-         coreMonitorBundle.wren,
+         Mux(coreMonitorBundle.wrenx, coreMonitorBundle.wrdata, 0.U),
```

Comment:
is there a plan to eventually enhance this trace format file to show wrenf wit the float result?

Reply:
yes: we do so in other uses of CoreMonitorBundle outside of Rocket-chip